mopidy_gmusic/__init__.py

```diff
@@ -21,7 +21,6 @@ def get_default_config(self):
     def get_config_schema(self):
         schema = super(GMusicExtension, self).get_config_schema()
 
-        schema['initial_code'] = config.Secret(optional=True)
```

Comment:
This should not be removed, as that will crash any Mopidy installation that has the value set in its configuration file. Instead, change from `config.Secret(...)` to `config.Deprecated()`, and Mopidy will ignore the value when present in the config file.

Reply:
Is this really an issue if there was never an actual release with that setting?

Reply:
No, in that case, I don't think this is an issue at all.